docs/Installation_Alternative.rst

```diff
@@ -36,13 +36,13 @@ you can use one of the following simple commands:
 
 -  using ``easy_install`` (old tool, but still works)::
 
-       easy_install --prefix /tmp easybuild
+       easy_install --prefix /tmp/$USER easybuild
```

Comment:
If you feel strongly about it, I'd go with something like `$HOME/eb` or `$HOME/EasyBuild` or somesuch. Anything other than `$HOME/.local` should be fine in terms of Python magic.